server/config.go

```diff
@@ -121,3 +181,57 @@ func (s *Service) ReplaceConfigSection(w http.ResponseWriter, r *http.Request) {
 
 	encodeJSON(w, http.StatusOK, res, s.Logger)
 }
+
+func validLogViewerUIConfig(cfg chronograf.LogViewerUIConfig) error {
+	if len(cfg.Columns) == 0 {
+		return fmt.Errorf("Invalid log viewer UI config: must have at least 1 column")
+	}
+
+	nameMatcher := map[string]bool{}
+	positionMatcher := map[int32]bool{}
+
+	for _, clm := range cfg.Columns {
+		iconCount := 0
+		textCount := 0
+		visibility := 0
+
+		if nameMatcher[clm.Name] == true {
```

Comment:
The idiomatic way to see if a key is set in a map in go is to do
```go
if _,ok:=nameMatcher[clm.Name]; ok {
}
```
See https://stackoverflow.com/questions/2050391/how-to-check-if-a-map-contains-a-key-in-go for details.